roles/auth_proxy/tasks/main.yml

```diff
@@ -1,6 +1,18 @@
 ---
 # tasks file for auth_proxy
 
+# Load the auth-proxy-image from local tar. Ignore any errors to handle the
+# case where the image is not built in
+- name: copy auth-proxy image
+  copy: src={{ auth_proxy_binaries }}/auth-proxy-image.tar dest=/tmp/auth-proxy-image.tar
+  when: auth_proxy_local_install == True
+  ignore_errors: yes
```

Comment:
@jojimt, Done. The reason for ignore_errors was that all of these had fall back alternatives. If docker load or docker pull fail, the service can try to pull or complete the pull at a later time when we do a docker run. I do see your point that failing fast is good for cases where these really dont work, for example a bad image name etc.